wp-modules/app/js/src/hooks/useEnvironment.ts

```diff
@@ -0,0 +1,25 @@
+import { useState } from '@wordpress/element';
+import { patternManager } from '../globals';
+import getHeaders from '../utils/getHeaders';
+
+export default function useEnvironment( initialEnvironmentNotice: boolean ) {
+	const [ displayNotice, setDisplayNotice ] = useState(
```

Comment:
Nice hook.

The state is very simple. Just `Boolean` state, and a handler to update it.